### DISCUSSION TITLE
Translation to Spanish

### DISCUSSION THREAD

**Discussion by jgonzalezalbisu** [TheOdinProject/theodinproject#3575] [Ideas]
Hi!
Has someone offered to contribute to translate the whole curriculum to Spanish? If not, how can I collaborate?

**Comment by KevinMulhern** [comment]
Hi @jgonzalezalbisu, thank you for the suggestion. 

Translating the curriculum to different languages is a request we get often. However, it is currently not on our roadmap. We provide some reasons here: https://github.com/TheOdinProject/blog/wiki/What-about-translations-of-your-curriculum

**Reply by jgonzalezalbisu** [reply]
Hi @KevinMulhern , thanks for your response I really appreciate it.
I imagined that this was probably a suggestion your team would get a lot, but it was worth a shot. I understand the implications of having to translate TOP to another language and the main reasons why this would be a problem and difficult to maintain by reading this [blog](https://github.com/TheOdinProject/blog/wiki/What-about-translations-of-your-curriculum); although I disagree on some points, I totally understand.

Again thanks for you response and your time!
